tests/schema/test_custom_scalar.py

```diff
@@ -85,10 +81,3 @@ def my_str(self, arg: MyStr) -> MyStr:
 
     assert not result.errors
     assert result.data["myStr"] == "valueSuffix"
-
-
-def test_error_when_registering_duplicate_scalar():
```

Comment:
what happens now if we create a scalar with the same name twice?

Reply:
Two situations to consider here:
1. Two scalars with same name for two _different_ schemas
2. Two scalars with same name for the _same_ schema

Reply:
> what happens now if we create a scalar with the same name twice?

I (think) the schema creation would fail in the same way that it would fail if there were 2 object types that had the same name. Will add a PR to verify it though.